lib/services/project-service.ts

```diff
@@ -140,7 +140,8 @@ export class ProjectService implements IProjectService {
 
 			shelljs.cp('-R', path.join(appSourcePath, "*"), appDestinationPath);
 			// Copy hidden files.
-			shelljs.cp('-R', path.join(appSourcePath, ".*"), appDestinationPath);
+			//TODO: plamen5kov: don't copy hidden files, until we find a reason to do it
+			// shelljs.cp('-R', path.join(appSourcePath, ".*"), appDestinationPath);
```

Comment:
Strongly disagree here. Why you need a reason for already existing code? Either remove it or keep it in the way it works today. Also I can reverse the question - why wouldn't you copy all files from template? Hidden files could be part of the template, so... why not copying them? And last - on Windows they are not hidden at all.